app/controllers/users_controller.rb

```diff
@@ -8,21 +8,60 @@ def show
   end
 
   def update
-    if @user.update(user_params)
-      redirect_to @user, notice: 'The user has been updated correctly.'
+    pass_params = password_params.to_h
+    exist_pass_arg = pass_params.any? { |k, v| !v.empty? and !v.nil? }
```

Comment:
This variable can be a private method what do you think?

Reply:
sure